Plugin/src/main/java/com/gmail/filoghost/holographicdisplays/bridge/protocollib/current/DebugHelper.java

```diff
@@ -59,7 +59,7 @@ private static String getPacketDescription(PacketContainer packetContainer) thro
 		// Get the first Minecraft super class
 		while (clazz != null && clazz != Object.class &&
 				(!MinecraftReflection.isMinecraftClass(clazz) || 
-				 Factory.class.isAssignableFrom(clazz))) {
+				 ByteBuddyGenerated.class.isAssignableFrom(clazz))) {

```

Comment:
Seems to be the same code as in [CommandPacket](https://github.com/dmulloy2/ProtocolLib/blob/master/src/main/java/com/comphenix/protocol/CommandPacket.java#L459), might be worth making that a public API?

Reply:
Yeah, that'd be a bit cleaner. Thanks for the commit!